index.html

```diff
@@ -28,6 +26,149 @@ <h2>I'am a Software developer</h2>
+  <img src="image/line.png" alt="">
+  </div>
+  <section class="fram-two">
+  <div class="featured-image">
+  <img src="image/Img-Placeholder.png" alt="">
+  </div>
+  <section class="right-block">
+  <h6>Multi-Post Stories</h6>
+  <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+  <section class="tags">
+  <img src="image/css.png" alt="">
+  <img src="image/html.png" alt="">
+  <img src="image/bootstrap.png" alt="">
+  <img src="image/ruby.png" alt=""> <br>
+  <button>See Project</button>
+  </section>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  </section>
+  <section class="frame43">
+  <section class="mask-group-one">
+  <h1>
+  Profesional Art Printing Data
+  </h1>
+  <p>
+  A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+  </p>
+  <section class="btn-group">
+  <button>html</button>
+  <button>bootstrap</button>
+  <button>ruby</button>
+  </section>
+  <section class="bottom" >
+  <p>See Project</p>
+  </section>
+  <section class="rectangle">
+  <h1>About me</h1>
```

Comment:
- [ ] (OPTIONAL) Kindly use an `h2` tag here as your site should have only one `h1` tag which should be the main heading for best semantic practices 💯